.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -82,8 +84,10 @@ jobs:
       - name: Get inputs or use defaults
         id: extract-inputs
         run: |
+          echo "build-branch=${{ inputs.build-branch                || false }}" >> "$GITHUB_OUTPUT"
           echo "branch=${{ inputs.branch                            || 'develop' }}" >> "$GITHUB_OUTPUT"
-          echo "delete-bee=${{ inputs.delete-bee                    || false }}" >> "$GITHUB_OUTPUT"
+          echo "bee-version-template=${{ inputs.bee-version-template || 'dev' }}" >> "$GITHUB_OUTPUT"
+          echo "delete-bee=${{ inputs.delete-bee                    || true }}" >> "$GITHUB_OUTPUT"
```

Comment:
Changing default value of delete-bee` to `true` for consistency with input documentation and usual use cases.